src/CMakeLists.txt

```diff
@@ -37,10 +37,6 @@ target_include_directories(polyMPO-core
   $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/test>
   $<INSTALL_INTERFACE:include>)
 target_link_libraries(polyMPO-core PUBLIC pumipic::pumipic)
-if(${NetCDF_FOUND})
-  target_compile_definitions(polyMPO-core PRIVATE POLYMPO_HAS_NETCDF)
-  target_link_libraries(polyMPO-core PRIVATE PkgConfig::NetCDF)
-endif()
 
```

Comment:
I think the above lines, and possibly the `target_include_directories(... ${NetCDF_Fortran_INCLUDEDIR})` command, should be added to the linking of executables in `test/CMakeLists.txt`.  IIRC, the tests depend on netcdf but not the library.

Reply:
ok added